react/ProductSummaryContext.tsx

```diff
@@ -115,11 +115,11 @@ const buildProductQuery = ((product: Product) => {
   return querystring.stringify(query)
 })
 
-function ProductSummaryProvider({ product, selectedItem, children }) {
+function ProductSummaryProvider({ product, selectedItem, isLoading, children }) {
   const initialState = {
     product,
     isHovering: false,
-    isLoading: false,
+    isLoading: Boolean(isLoading),
```

Comment:
just a suggestion, once I don't know if you really need the `Boolean()` 🤔 

```suggestion
    isLoading: isLoading ?? false,
```

Reply:
It was not necessary 🤦